crates/jstz_cli/src/config.rs

```diff
@@ -136,51 +189,31 @@ pub struct SandboxConfig {
     pub octez_node_dir: PathBuf,
     /// Directory of the octez rollup node
     pub octez_rollup_node_dir: PathBuf,
-    /// The port of the octez node
-    pub octez_node_port: u16,
-    /// The port of the octez RPC node
-    pub octez_node_rpc_port: u16,
-    /// The port of the jstz node
-    pub jstz_node_port: u16,
     /// Pid of the pid
     pub pid: u32,
-    private: (),
 }
 
-impl SandboxConfig {
-    pub fn new(
-        pid: u32,
-        octez_client_dir: PathBuf,
-        octez_node_dir: PathBuf,
-        octez_rollup_node_dir: PathBuf,
-    ) -> Self {
-        Self {
-            octez_client_dir,
-            octez_node_dir,
-            octez_rollup_node_dir,
-            pid,
-            octez_node_port: 18731,
-            octez_node_rpc_port: 18730,
-            jstz_node_port: 8933,
-            private: (),
-        }
-    }
-}
+pub const SANDBOX_OCTEZ_NODE_PORT: u16 = 18731;
+pub const SANDBOX_OCTEZ_NODE_RPC_PORT: u16 = 18730;
+pub const SANDBOX_JSTZ_NODE_PORT: u16 = 8933;
+pub const SANDBOX_OCTEZ_SMART_ROLLUP_PORT: u16 = 8932;
 
 impl Config {
     /// Path to the configuration file
     pub fn path() -> PathBuf {
-        home().join("config.json")
+        jstz_home_dir().join("config.json")
     }
 
     /// Load the configuration from the file
-    pub fn load() -> std::io::Result<Self> {
+    pub fn load() -> Result<Self> {
         let path = Self::path();
 
         let config = if path.exists() {
             let json = fs::read_to_string(&path)?;
-            serde_json::from_str(&json)
-                .map_err(|e| Error::new(ErrorKind::InvalidData, e))?
+
+            serde_json::from_str(&json).map_err(|_| {
+                user_error!("Your configuration file is improperly configured.")
```

Comment:
Would be good to write some instructions about how to configure it.

Reply:
Currently we don't have a docs link for config. Once we do, we can link it in this user error.